Number Verifier/Sdl.Community.NumberVerifier.Tests/OmitZero/OmitZeroSourceAndTarget.cs

```diff
@@ -134,32 +133,29 @@ public void SourceOmitCheckedTargetUncheckedNoError(string source, string target
         public void SourceOmitCheckedTargetUncheckedError(string source, string target)
         {
             var errorMessage = SourceOmitCheckedTargetUnchecked(source, target);
-            Assert.Equal(errorMessage[0].ErrorMessage, "Number(s) modified/unlocalised. ");
-
+            Assert.Equal("Number removed. ", errorMessage[0].ErrorMessage);
         }
 
         [Theory]
         [InlineData("-.55", "-.55")]
-        public void SourceOmitCheckedTargetUncheckedNegativeNumbers(string source, string target)
+        public void SourceOmitCheckedTargetUncheckedNegativeNumbersNoError(string source, string target)
```

Comment:
Can we have a unit test which tests the error message?

Reply:
Yes, an UT which test the error message was added in the new commit
![image](https://user-images.githubusercontent.com/13149323/82298449-d08ca100-99bc-11ea-937e-62dd62a3ce0e.png)